### PR TITLE
Convert The Block List Project Lists to domains

### PR DESCRIPTION
**Pull request by GiddyGoatGaming**

### CODE REVIEW

**Comment by ignoramous** [comment]
This list is missing:

```
https://blocklistproject.github.io/Lists/alt-version/facebook-nl.txtt
```

**Comment by GiddyGoatGaming** [comment]
I put that in from what from what my commit is showing here. https://blocklistproject.github.io/Lists/alt-version/facebook-nl.txtt right? Just below the piracy list

File: config.json [hunk 1]
```diff
@@ -1428,82 +1428,82 @@
...
       "pack": ["piracy", "file-hosts"]
     },
     {
       "vname": "Facebook (The Block List Project)",
-      "format": "hosts",
+      "format": "domains",
       "group": "ParentalControl",
       "subg": "The Block List Project",
-      "url": "https://blocklistproject.github.io/Lists/facebook.txt",
+      "url": "https://blocklistproject.github.io/Lists/alt-version/facebook-nl.txtt",
```
Comment (ignoramous) [inline config.json:1498]: pls drop the extra `t` from `.txtt`.

the current url is 404.

Comment (GiddyGoatGaming) [reply config.json]: Wow, that’s embarrassing. How did I not realize that? Im so sorry😂

Comment (GiddyGoatGaming) [reply config.json]: It should now return a valid address

**Review by ignoramous — APPROVED** [review]

**Comment by ignoramous** [comment]
Thanks!

**Pull request merged**